airlock/management/commands/create_release_request.py

```diff
@@ -65,6 +66,19 @@ def add_arguments(self, parser):
     def handle(self, username, workspace_name, **options):
         user = User.objects.get(user_id=username)
         request = bll.get_or_create_current_request(workspace_name, user)
+        if request.is_under_review():
+            self.stdout.write(
+                f"A release request for workspace '{workspace_name}' is already under review for user '{username}'"
+            )
+            # Management commands expect to write their output to std out, so return it as a (json)string
```

Comment:
Huh, I thought you had to print it yourself, and returning anything was only for internal use and testing.

Reply:
TIL :D